tests/sfbase.py

```diff
@@ -0,0 +1,937 @@
+"""
```

Comment:
I am not up to speed on these split taps with two base files yet.  Will save this one for Monday unless you and Jyothi can get me up to speed.

Reply:
Sure, basically copied over base.py and 
instead of 
from tap_tester.base_case import BaseCase
have this: from tap_tester.base_suite_tests.base_case import BaseCase
- Removed unnecessary methods 
- commented out some fields in metadata that is not discovered by discovery to make the assertion pass for discovery

Reply:
So is the plan to maintain two base files until the full suite is cut over then remove / rename one?  Why not just extend / modify one base file?

Reply:
yeah, the idea is to have 2 files still fuly cutover.
Not sure if it is possible to use the same file as it can create conflicts since there are some methods in both tap_tester.base_case and also tap_tester.base_suite_tests.base_case